sql/queries.sql

```diff
@@ -110,12 +110,27 @@ CREATE TABLE authors ( name text, id bigint );
 WITH long_names AS ( SELECT id FROM authors WHERE char_length(name) > 15 )
 SELECT title FROM articles;
 
--- joins are unsupported
-SELECT title, authors.name FROM authors, articles WHERE authors.id = articles.author_id;
+--  queries which involve functions in FROM clause are unsupported.
+select * from  articles, position('om' in 'Thomas');
 
--- subqueries are not supported
+-- subqueries are not supported in WHERE clause
 SELECT * FROM articles WHERE author_id IN (SELECT id FROM authors WHERE name LIKE '%a');
 
+-- subqueries are not supported in FROM clause
+SELECT articles.id,test.word_count FROM articles, (SELECT id, word_count FROM articles) AS test where test.id = articles.id;
+
+-- subqueries are not supported in SELECT clause
+SELECT  a.title AS  name,(SELECT a2.id FROM authors a2 WHERE a.id = a2.id  LIMIT 1) AS special_price FROM articles a;
+
+-- joins are not supported in WHERE clause
+SELECT title, authors.name FROM authors, articles WHERE authors.id = articles.author_id;
+
+-- joins are not supported in FROM clause
+SELECT * FROM  (articles INNER JOIN authors ON articles.id = authors.id);
+
+
```

Comment:
If this space is intentional (and intended to separate the file into single- and multi-shard tests), make it two blank lines. We don't use any more than that in our style.